src/stores/FiltersHistoryStore.ts

```diff
@@ -0,0 +1,79 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import { observable, computed, action } from 'mobx';
+import { IndexedDB, IndexedDbStores, indexedDbLimits } from '../api/indexedDb';
+import { SearchPanelType } from '../components/search-panel/SearchPanel';
+import {
+	EventFilterState,
+	MessageFilterState,
+	FilterState,
+} from '../components/search-panel/SearchPanelFilters';
+import { getEquilizedFilterState } from '../helpers/search';
+
+export interface FiltersHistory {
+	timestamp: number;
+	type: SearchPanelType;
+	filters: Partial<EventFilterState> | Partial<MessageFilterState>;
+}
+
+class FiltersHistoryStore {
+	constructor(private indexedDb: IndexedDB) {
+		this.init();
+	}
+
+	@observable
+	public history: FiltersHistory[] = [];
```

Comment:
This store stores same data as EventFilterAutocompleteStore and MessagesFilterAutocompleteStore.

Some store is redundant.